dask/distributed/worker.py

```diff
@@ -130,6 +136,10 @@ def __init__(self, scheduler, data=None, nthreads=100,
         self._listen_scheduler_thread.start()
         self._listen_workers_thread = Thread(target=self.listen_to_workers)
         self._listen_workers_thread.start()
+        self._heartbeat_thread = Thread(target=self.heartbeat,
+                                        kwargs={'pulse': heartbeat_pulse})
+        self._heartbeat_thread.event = Event()
+        self._heartbeat_thread.start()
```

Comment:
or `if not self.heartbeat` if we expect users might use `False` which seems reasonable.